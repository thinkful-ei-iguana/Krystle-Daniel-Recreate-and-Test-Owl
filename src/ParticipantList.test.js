import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import ParticipantList from './ParticipantsList';


describe('Participant List', () => {
  it('renders without crashing', () => {
    const div = document.createElement('div');
    ReactDOM.render(<ParticipantList />, div);
    ReactDOM.unmountComponentAtNode(div);
  });

  it('renders the UI as expected', () => {
    const tree = renderer
      .create(<ParticipantList/>)
      .toJSON();
    expect(tree).toMatchSnapshot();  
  });

 
});