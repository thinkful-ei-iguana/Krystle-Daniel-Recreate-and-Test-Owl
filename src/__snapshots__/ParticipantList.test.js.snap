// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Participant List renders the UI as expected 1`] = `
<div
  className="participants-list"
>
  <div
    className="participant"
  >
    <div
      className="participant-avatar"
    >
      <img
        src="https://robohash.org/itaquevoluptatumrerum.jpg?size=200x200&set=set1"
      />
    </div>
    <div
      className="participant-info"
    >
      <div
        className="participant-name"
      >
        Koren Templeton
      </div>
      <div
        className="participant-status"
      >
        on stage
      </div>
    </div>
  </div>
  <div
    className="participant"
  >
    <div
      className="participant-avatar"
    >
      <img
        src="https://robohash.org/doloredolorescupiditate.jpg?size=200x200&set=set1"
      />
    </div>
    <div
      className="participant-info"
    >
      <div
        className="participant-name"
      >
        Caty Flucker
      </div>
      <div
        className="participant-status"
      >
        left session
      </div>
    </div>
  </div>
  <div
    className="participant"
  >
    <div
      className="participant-avatar"
    >
      <img
        src="https://robohash.org/utlaborumfugit.jpg?size=200x200&set=set1"
      />
    </div>
    <div
      className="participant-info"
    >
      <div
        className="participant-name"
      >
        Axe Kubicka
      </div>
      <div
        className="participant-status"
      >
        left session
      </div>
    </div>
  </div>
  <div
    className="participant"
  >
    <div
      className="participant-avatar"
    >
      <img
        src="https://robohash.org/etexercitationemassumenda.jpg?size=200x200&set=set1"
      />
    </div>
    <div
      className="participant-info"
    >
      <div
        className="participant-name"
      >
        Frank Runciman
      </div>
      <div
        className="participant-status"
      >
        left session
      </div>
    </div>
  </div>
  <div
    className="participant"
  >
    <div
      className="participant-avatar"
    >
      <img
        src="https://robohash.org/iustodoloremqueinventore.jpg?size=200x200&set=set1"
      />
    </div>
    <div
      className="participant-info"
    >
      <div
        className="participant-name"
      >
        Ashla Attwool
      </div>
      <div
        className="participant-status"
      >
        on stage
      </div>
    </div>
  </div>
</div>
`;
